test/unit/uk/gov/hmrc/tariffclassificationfrontend/connector/BindingTariffClassificationConnectorSpec.scala

```diff
@@ -41,13 +44,13 @@ class BindingTariffClassificationConnectorSpec extends UnitSpec with WiremockTes
   private val otherQueue = Queue(2, "other", "Other")
   private implicit val hc = HeaderCarrier()
 
+  given(configuration.bindingTariffClassificationUrl).willReturn("http://localhost:20001")
```

Comment:
Can this be in a before block rather than here?

Reply:
WiremockTestServer is already implementing this before each... then we will need to think in other strategies.... I will revert it back